### ISSUE TITLE
UserModInstances - triggers crash in DDP and Artnet modules

### ISSUE THREAD

**Issue by netmindz** [ewoudwijma/StarMod#33]
Detection of WLED instances triggers crash in setup of DDP and ArtNet

```
if (packetSize == sizeof(UDPWLEDMessage)) { //WLED instance
...
          // updateNode(starModMessage);
        }
```

Commenting updateNode prevents the crash

**Comment by netmindz** [comment]
Backtrace: 0x40083a75:0x3ffb1380 0x4008dfa9:0x3ffb13a0 0x40082e2e:0x3ffb13c0 0x400ed8a5:0x3ffb13e0 0x400ed8ce:0x3ffb1470 0x400d82c5:0x3ffb1490 0x400d8506:0x3ffb14c0 0x400eb4ff:0x3ffb1500 0x400f0e3a:0x3ffb1530 0x400f7328:0x3ffb1630 0x400f749a:0x3ffb2230 0x400e7c35:0x3ffb2250 0x400eaab0:0x3ffb2270 0x40109f55:0x3ffb2290

  #0  0x40083a75:0x3ffb1380 in panic_abort at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/panic.c:408
  #1  0x4008dfa9:0x3ffb13a0 in esp_system_abort at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/esp_system.c:137
  #2  0x40082e2e:0x3ffb13c0 in __stack_chk_fail at /Users/ficeto/Desktop/ESP32/ESP32S2/esp-idf-public/components/esp_system/stack_check.c:35
  #3  0x400ed8a5:0x3ffb13e0 in UserModDDP::setup()::{lambda(ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char)#1}::operator()(ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char) const at src/User/UserModDDP.h:82
  #4  0x400ed8ce:0x3ffb1470 in std::_Function_handler<unsigned char (ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char), UserModDDP::setup()::{lambda(ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char)#1}>::_M_invoke(std::_Any_data const&, ArduinoJson::V703PB2::JsonObject&&, unsigned char&&, ArduinoJson::V703PB2::JsonObject&&) at /home/will/.platformio/packages/toolchain-xtensa-esp32/xtensa-esp32-elf/include/c++/8.4.0/bits/std_function.h:282
  #5  0x400d82c5:0x3ffb1490 in std::function<unsigned char (ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char)>::operator()(ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char) const at /home/will/.platformio/packages/toolchain-xtensa-esp32/xtensa-esp32-elf/include/c++/8.4.0/bits/std_function.h:687
  #6  0x400d8506:0x3ffb14c0 in SysModUI::callVarFun(ArduinoJson::V703PB2::JsonObject, unsigned char, unsigned char) at src/Sys/SysModUI.h:224
  #7  0x400eb4ff:0x3ffb1500 in SysModUI::callVarFun(char const*, unsigned char, unsigned char) at src/Sys/SysModUI.h:215
  #8  0x400f0e3a:0x3ffb1530 in UserModInstances::updateNode(UDPStarModMessage) at src/User/UserModInstances.h:674
  #9  0x400f7328:0x3ffb1630 in UserModInstances::handleNotifications() at src/User/UserModInstances.h:470
  #10 0x400f749a:0x3ffb2230 in UserModInstances::loop() at src/User/UserModInstances.h:387
  #11 0x400e7c35:0x3ffb2250 in SysModules::loop() at src/SysModules.cpp:101
  #12 0x400eaab0:0x3ffb2270 in loop() at src/main.cpp:133
  #13 0x40109f55:0x3ffb2290 in loopTask(void*) at /home/will/.platformio/packages/framework-arduinoespressif32/cores/esp32/main.cpp:50

**Comment by ewoudwijma** [comment]
Try latest commit as that includes a char out of bounds bugfix which might have caused this indirectly

**Comment by netmindz** [comment]
Fixed in 4538cc2252190b54e3c05db47f8b9fec46942cab
